code/controllers/subsystem/daynightcycle.dm

```diff
@@ -0,0 +1,77 @@
+var/datum/subsystem/daynightcycle/SSDayNight
+
+#define TOD_MORNING 	1
```

Comment:
since these values are arbitrary, why not just define them by their color? e.g.: 
`#define TOD_MORNING #002235` - you could even eliminate a variable this way because no need to track color separately

Reply:
Removed the currentcolor var